.bazelrc

```diff
@@ -76,10 +76,6 @@ build:tsan-dev --copt -DEVENT__DISABLE_DEBUG_MODE
 # https://github.com/google/sanitizers/issues/953
 build:tsan-dev --test_env="TSAN_OPTIONS=report_atomic_races=0"
 
-# Exclude debug info from the release binary since it makes it too large to fit
-# into a zip file. This shouldn't affect crash reports.
-build:release-common --define=no_debug_info=1
```

Comment:
need to change to use for ios but not both instead, because the download size is an issue otherwise

Reply:
OK, that makes sense